src/odemis/gui/cont/fastem_acq.py

```diff
@@ -564,12 +548,11 @@ def on_align(self, evt):
                                self._main_data_model.beamshift, self._main_data_model.det_rotator,
                                calibrations=self.calibrations)
 
-        f.add_done_callback(self._on_alignment_done)  # also handles cancelling and exceptions
+        f.add_done_callback(self._on_calibration_done)  # also handles cancelling and exceptions
         # connect the future to the progress bar and its label
-        self._future_connector = ProgressiveFutureConnector(f, self.gauge_progress, self.gauge_label)
+        self._future_connector = ProgressiveFutureConnector(f, self.gauge, self.label, full=False)
 
-    @call_in_wx_main
-    def _on_alignment_done(self, future):
+    def _on_calibration_done(self, future, _=None):
```

Comment:
I added @call_in_wx_main above this function

Reply:
Thanks. I am surprised that it needed to be added here as well. I understood that I only need to add it if something is adjusted in the GUI. Is this because I call here a function that eventually adjusts something in the GUI?